californium-core/src/main/java/org/eclipse/californium/core/network/InMemoryMessageExchangeStore.java

```diff
@@ -271,35 +258,24 @@ public boolean registerOutboundRequestWithTokenOnly(final Exchange exchange) {
 
 	@Override
 	public void remove(final KeyToken token, final Exchange exchange) {
-		boolean removed;
-		int size;
-		synchronized (exchangesByToken) {
-			removed = exchangesByToken.remove(token, exchange);
-			size = exchangesByToken.size();
-		}
+		boolean removed = exchangesByToken.remove(token, exchange);
 		if (removed) {
-			LOGGER.log(Level.FINE, "removing exchange for token {0}, remaining exchanges by tokens: {1}",
-					new Object[] { token, size });
+			LOGGER.log(Level.FINE, "removing exchange for token {0}", new Object[] { token });
```

Comment:
you generously removed the information about the remaining number of exchanges by token, is that information logged elsewhere now or is this the price we have to pay for getting rid of the synchronization?